app/views/ai-league/AILeagueView.vue

```diff
@@ -40,6 +40,7 @@ export default {
   },
 
   mounted () {
+    this.fetchData({ componentName: this.$options.name, options: { loadedEventName: 'AI Leauge: Loaded' } })
```

Comment:
what happened before? Why do we have to fetch data for dropdown? Also, what's with typos in league? 😅

Reply:
The problem occurs because changing to the ai-league view does not change the component name so the top bar is not updated. fetchData is used just to update the component name.